group_vars/data.yml

```diff
@@ -7,17 +7,32 @@ thredds:
     roles: "tdrAdmin,tdsConfig"
 
 thredds_webapp:
-  url: "{{ mirror.base }}/2.8/1/thredds/5.0/5.0.3/thredds.war"
+  url: "{{ assets.base }}/thredds.war"
   name: thredds
 
+stats_api_version: v1.0.9
+
 stats_webapp:
-  url: "{{ mirror.base }}/2.8/1/esgf-stats-api/esgf-stats-api.war"
+  url: "{{ assets.stats_api }}/esgf-stats-api.war"
   name: esgf-stats-api
 
+dashboard:
```

Comment:
Can you remove this whole dashboard block here. It conflicts with that in group_vars/all.yml